cmd/options/options_test.go

```diff
@@ -319,30 +322,36 @@ func TestSetConfigFromDeprecatedOptionsOrDie(t *testing.T) {
 			orig: NodeProblemDetectorOptions{
 				SystemLogMonitorConfigPaths: []string{"config-a"},
 				MonitorConfigPaths: types.ProblemDaemonConfigPathMap{
-					systemlogmonitor.SystemLogMonitorName: &[]string{"config-b"},
+					systemLogMonitorName: &[]string{"config-b"},
 				},
 			},
 			expectPanic: true,
-			wanted: NodeProblemDetectorOptions{
-				MonitorConfigPaths: types.ProblemDaemonConfigPathMap{
-					systemlogmonitor.SystemLogMonitorName: &[]string{"config-b"},
-				},
-			},
 		},
 		{
 			name: "using deprecated & new options on CustomPluginMonitor",
 			orig: NodeProblemDetectorOptions{
 				CustomPluginMonitorConfigPaths: []string{"config-a"},
 				MonitorConfigPaths: types.ProblemDaemonConfigPathMap{
-					custompluginmonitor.CustomPluginMonitorName: &[]string{"config-b"},
+					customPluginMonitorName: &[]string{"config-b"},
 				},
 			},
 			expectPanic: true,
-			wanted: NodeProblemDetectorOptions{
-				MonitorConfigPaths: types.ProblemDaemonConfigPathMap{
-					custompluginmonitor.CustomPluginMonitorName: &[]string{"config-b"},
-				},
+		},
+		{
+			name: "using deprecated options when SystemLogMonitor is not registered",
+			orig: NodeProblemDetectorOptions{
+				SystemLogMonitorConfigPaths: []string{"config-a"},
+				MonitorConfigPaths:          types.ProblemDaemonConfigPathMap{},
 			},
```

Comment:
nit: probably change to:
```
types.ProblemDaemonConfigPathMap{
    customPluginMonitorName: &[]string{},
}
```

And similar for below test case,

Reply:
Oh just realized that github does not highlight the selected code.
So to clarify:
I think we should change `MonitorConfigPaths:          types.ProblemDaemonConfigPathMap{},` into
```
MonitorConfigPaths: types.ProblemDaemonConfigPathMap{
    customPluginMonitorName: &[]string{},
},
```

Reply:
Not needed, because we are not using custom plugin monitor deprecated flag, it is fine to not register that plugin. :) This also helps us proves that.

Reply:
How about this?

Reply:
Thanks!